pywwt/nbextension/static/wwt_json_api.js

```diff
@@ -179,8 +179,37 @@ function wwt_apply_json_message(wwt, msg) {
       wwt.setForegroundImageByName(msg['mode']);
       break;
 
-    case 'track_object':
-      wwtlib.WWTControl.singleton.renderContext.set_solarSystemTrack(msg['code']);
+    case 'track_and_zoom':
+
+      var object = msg['obj'];
+      var instant = msg['inst'];
+
+      // Collect the arguments necessary to create a Place object
+      var ra = wwt.getRA();
+      var dec = wwt.getDec();
+      var classification = wwtlib.Classification.solarSystem;
+      var constellation;
+      var imageType = wwtlib.ImageSetType.sky;
+      //var imageType = wwtlib.ImageSetType.solarSystem;
+      var scale = wwt.settings.get_solarSystemScale();
+      var zoom; //= (.02 * scale + .08) / msg['zoom'];
+      // if imageType == wwtlib.ImageSetType.solarSystem, use the equation
+      // elif imageType == wwtlib.ImageSetType.sky, gotoTarget pre-sets zoom
```

Comment:
There is still some unused code here - can you remove anything that's not used?

Reply:
I put the extra code in a comment that describes a method we might use to give more flexibility to this function once we fix RA/Dec for planets. It would be nice to keep it handy so I don't forget what I found when searching through the SDK.